iiop-openjdk/src/main/resources/schema/jboss-as-iiop-openjdk_1_0.xsd

```diff
@@ -69,8 +70,9 @@
             </xs:documentation>
         </xs:annotation>
         <xs:attribute name="giop-version" type="xs:string" use="optional" default="1.2"/>
-        <xs:attribute name="socket-binding" type="xs:string" use="optional" default="iiop"/>
-        <xs:attribute name="ssl-socket-binding" type="xs:string" use="optional" default="iiop-ssl"/>
+        <xs:attribute name="socket-binding" type="xs:string" use="required" />
```

Comment:
I've reverted the socket binding config to be optional again with default values